feature/login-check/src/main/java/com/simprints/feature/logincheck/usecases/UpdateStoredUserIdUseCase.kt

```diff
@@ -0,0 +1,18 @@
+package com.simprints.feature.logincheck.usecases
+
+import com.simprints.core.domain.tokenization.TokenizableString
+import com.simprints.infra.authstore.AuthStore
+import javax.inject.Inject
+
+class UpdateStoredUserIdUseCase @Inject constructor(
+    private val authStore: AuthStore,
+) {
+
+    operator fun invoke(userId: TokenizableString) {
+        // This is a hack to make sure that there is a stored user ID if user logged in before 2024.1.0.
+        // It could be removed after all we drop support for older versions.
```

Comment:
Do you think there might be a need to check the SID version as well? If no, feel free to resolve

Reply:
It isn't needed explicitly. As long as there is a cached user ID we do not care about the exact version.